### ISSUE TITLE
Failed to arm not showing up as a notification anymore

### ISSUE THREAD

**Issue by xhemp** [nielsfaber/alarmo#20]
I'm using the latest 1.1.2, just wondering if there was any change on notifications (on purpose or not) as I haven't been able to see the failed to arm notifications anymore, other notifications look ok ie: If I try to arm the alarm with a sensor open, it doesn't work and it also doesn't show me anything even though I have a notification set.

**Comment by nielsfaber** [comment]
I will take a look. This is not intentional.

**Comment by nielsfaber** [comment]
I saw that a bug that causes no notification to be sent if:

1. You have a leave delay > 0, and
2. There is a sensor open, which is configured to trigger immediate

This is a special case since such sensors are evaluated at the start of the arming process, where others are evaluated at the end of the arming process (so when leave delay has expired).

I also added some logging for these scenario's.

I'm assuming that this scenario is what you were encountering.
For other scenario's I was not able to reproduce the issue.
The bug is patched and will be included in the upcoming release.
